### PR TITLE
Added webpack 2 example

### PR DESCRIPTION
**Pull request by mauricedb**
Added a Webpack 2 configuration example as the syntax is slightly different with `rules` instead of `loaders`.

### CODE REVIEW

File: pages/Integrating with Build Tools.md [hunk 1]
```diff
@@ -167,6 +167,27 @@ module.exports = {
...
+        path: '/',
+        filename: "bundle.js"
+    },
+    resolve: {
+        extensions: [".tsx", ".ts", ".js"]
+    },
+    module: {
+        rules: [
+            // all files with a '.ts' or '.tsx' extension will be handled by 'ts-loader'
+            { test: /\.tsx?$/, loader: "ts-loader", exclude: /node_modules/ }
```
Comment (TheLarkInn) [inline pages/Integrating with Build Tools.md]: Please change loader to use

File: pages/Integrating with Build Tools.md [hunk 2]
```diff
@@ -167,6 +167,27 @@ module.exports = {
...
+
+```js
+module.exports = {
+    entry: "./src/index.tsx",
+    output: {
+        path: '/',
+        filename: "bundle.js"
+    },
+    resolve: {
+        extensions: [".tsx", ".ts", ".js"]
```
Comment (TheLarkInn) [inline pages/Integrating with Build Tools.md]: This is an incorrect "appending" from the default values. See webpack.js.org/configuration/resolve

File: pages/Integrating with Build Tools.md (hunk 1 above)
Comment (DanielRosenwasser) [inline pages/Integrating with Build Tools.md]: For the record, I think @TheLarkInn is referring to https://webpack.js.org/configuration/module/#rule-use

File: pages/Integrating with Build Tools.md [hunk 3]
```diff
@@ -167,6 +167,27 @@ module.exports = {
 }
 ```
 
+### Basic webpack.config.js when using Webpack 2
```
Comment (DanielRosenwasser) [inline pages/Integrating with Build Tools.md]: Can you swap sections for Webpack 1 & 2?

**Comment by mauricedb** [comment]
@TheLarkInn and @DanielRosenwasser 

I have updated the PR with the requested changes. Please let me know if this is what you were looking for.

Also rebased the PR on master.

Maurice

**Comment by mauricedb** [comment]
@DanielRosenwasser Is this ever going to get merged? The docs still only show the Webpack 1  configuration and we are at version 4 already.

**Pull request merged**